src/Compilers/Core/Portable/Symbols/NullableAnnotation.cs

```diff
@@ -0,0 +1,15 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace Microsoft.CodeAnalysis
+{
+    // PROTOTYPE(nullable-api): Document
+    public enum NullableAnnotation : byte
+    {
+        NotApplicable = 0,
+        Disabled,      // No information. Think oblivious.
+        NotAnnotated, // Type is not annotated - string, int, T (including the case when T is unconstrained).
+        Annotated,    // Type is annotated - string?, T? where T : class; and for int?, T? where T : struct.
+        NotNullable,  // Explicitly set by flow analysis
+        Nullable,     // Explicitly set by flow analysis
```

Comment:
Can these be removed?

Reply:
Not yet. I'll remove them when I do the merge from master.

---
In reply to: [266097888](https://github.com/dotnet/roslyn/pull/34017#discussion_r266097888) [](ancestors = 266097888)